smpte.css

```diff
@@ -249,6 +273,21 @@ figure {
   white-space: nowrap;
 }
 
+#sec-normative-references .ext-ref::before {
```

Comment:
Why are normative references styled differently than bibliographic references?

Reply:
They are not different:

```
#sec-normative-references .ext-ref::before {
  white-space: pre;
  content: '\A';
}

#sec-bibliography .ext-ref::before {
  white-space: pre;
  content: '\A';
}
```

Reply:
Updated via https://github.com/SMPTE/html-pub/pull/27/commits/1d1c2d8b590b01bba6c7d31eb9773b5271602c91